containers.go

```diff
@@ -0,0 +1,46 @@
+package proxmox
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+func (c *Container) Start(client *Client, node string) (status ContainerStatus, err error) {
+	return status, client.Post(fmt.Sprintf("/nodes/%s/lxc/%s/status/start", node, c.VMID), []byte{}, &status)
+}
+
+func (c *Container) Stop(client *Client, node string) (status ContainerStatus, err error) {
+	json, err := json.Marshal(c)
```

Comment:
starting to find this repeated a ton, maybe we should offer Post and Put to take an interface{} and marshal it fore you on the way through?

Reply:
we will do this later, this can merge as is as it's obviously functional so resolving my comment https://github.com/luthermonson/go-proxmox/issues/4